target/linux/realtek/dts/rtl9312_plasmacloud_psx28.dts

```diff
@@ -0,0 +1,700 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+/dts-v1/;
+
+#include "rtl931x.dtsi"
+
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/leds/common.h>
+
+#define ENABLE_I2C
+#define ENABLE_I2C_MUX
+#define ENABLE_SFF_SFP
+#define ENABLE_SFP_PORTS
+#define ENABLE_SFP_PHYS
+// #define IN_BAND_STATUS
+
+/ {
+	compatible = "plasmacloud,psx28", "plasmacloud,esx28", "rapidmax,psx24", "realtek,rtl838x-soc";
+	model = "Plasma Cloud PSX28";
+
+	chosen {
+		bootargs = "earlycon loglevel=8 nowait";
+	};
+
+	gpio-restart {
+		compatible = "gpio-restart";
+		gpios = <&gpio0 31 GPIO_ACTIVE_LOW>;
+	};
+
+	keys {
+		compatible = "gpio-keys-polled";
+		poll-interval = <20>;
+
+		mode {
+			label = "reset";
+			gpios = <&gpio1 3 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+	};
+
+	gpio1: rtl8231-gpio {
+		compatible = "realtek,rtl8231-gpio";
+		#gpio-cells = <2>;
+		gpio-controller;
+		indirect-access-bus-id = <24>;
+	};
```

Comment:
This is outdated and needs to be converted. Run

```
git log -p -Srealtek,rtl8231-gpio origin/main -- target/linux/realtek/dts/
```

to see a couple of examples